pkgs/development/tools/misc/sipp/default.nix

```
@@ -1,29 +1,34 @@
-{lib, stdenv, fetchurl, ncurses, libpcap }:
+{ lib, stdenv, fetchurl, ncurses, libpcap, cmake, openssl, git, lksctp-tools }:
 
 stdenv.mkDerivation rec {
-  version = "3.6.0";
-
+  version = "3.6.1";
   pname = "sipp";
 
   src = fetchurl {
     url = "https://github.com/SIPp/${pname}/releases/download/v${version}/${pname}-${version}.tar.gz";
-    sha256 = "1fx1iy2n0m2kr91n1ii30frbscq375k3lqihdgvrqxn0zq8pnzp4";
+    sha256 = "sha256-alYOg6/5gvMx3byt+zvVMMWJbNW3V91utoITPMhg7LE=";
   };
 
   postPatch = ''
-    sed -i "s@pcap/\(.*\).pcap@$out/share/pcap/\1.pcap@g" src/scenario.cpp
+    cp version.h src/version.h
   '';
 
-  configureFlags = [
-    "--with-pcap"
-  ];
-
-  postInstall = ''
-    mkdir -pv $out/share/pcap
-    cp pcap/* $out/share/pcap
+  installPhase = ''
+    mkdir -p $out/bin
+    mv sipp $out/bin
   '';
 
-  buildInputs = [ncurses libpcap];
+  buildInputs = [ ncurses libpcap openssl ]
+    ++ lib.optionals (stdenv.isLinux) lksctp-tools;
```

```suggestion
    ++ lib.optional (stdenv.isLinux) lksctp-tools;
```

You changed the semantic from list to package, so you need to use `optional` instead of `optionals`.